tests/Platforms/OraclePlatformTest.php

```diff
@@ -931,4 +931,24 @@ protected function getLimitOffsetCastToIntExpectedQuery(): string
         return 'SELECT * FROM (SELECT a.*, ROWNUM AS doctrine_rownum FROM (SELECT * FROM user) a WHERE ROWNUM <= 3)'
             . ' WHERE doctrine_rownum >= 3';
     }
+
+    public function testGetSubstringExpression(): void
+    {
+        self::assertSame(
+            'SUBSTR(\'foobar\', 4, 6)',
```

Comment:
Let's avoid escaping, it's harder to read IMO:

```suggestion
            "SUBSTR('foobar', 4, 6)",
```

Reply:
I agree.